banking-app/demo.sh

```diff
@@ -2,90 +2,15 @@
 set -euox pipefail
 
 # Run `make start-host` in another terminal before running this script
+# To build and start a ccf network and automatically deploy your application on it.
 
-
+# Change working directory to the one that contains 
+# proposals, generated certs and keys, and constitution files
 cd workspace/sandbox_common
```

Comment:
nit: Personally preference, but if a script changes directories, I always want it to ensure that I get left where I started from. Especially if this gets called from someone else. So maybe try something like this instead of the cd
```bash
pushd "workspace/sandbox_common" > /dev/null

# reset the current directory on exit using a trap so that the directory is reset even on error
function finish {
  popd > /dev/null
}
trap finish EXIT
```

Reply:
@ross-p-smith  Just to confirm, from user perspective, it doesn't change anything, is it correct?
```bash
mkdir test
echo "cd test && pwd" > test.sh && chmod +x test.sh
./test.sh # output: <working directory>/test
pwd # output: <working directory>
```

Reply:
Spoken with @Aymalla - I'm happy that it doesn't leave us in an incorrect directory